### ISSUE TITLE
WARNING:root:unhandled entity type SPLINE

### ISSUE THREAD

**Issue by Jemu1** [goulu/Goulib#6]
When attempting to draw a simple spline I get the message:
WARNING:root:unhandled entity type SPLINE

Circle and lines are normally plotted.

![untitled-1 copy](https://cloud.githubusercontent.com/assets/24817498/21533924/434cee3e-cd60-11e6-8cb6-3a35dc57d0d2.JPG)

Line used: Drawing("C:/Users/User1/Desktop/Draw1.DXF").

**Comment by goulu** [comment]
yeah, well... splines aren't fully implemented at the moment. only older "cubic splines" are handled in Goulib.drawing , newer "Bezier splines" aren't yet. 

You're most welcome to contribute to Goulib if you want to add this feature (there is a commented out stub of code with ref urls in drawing. py to start with) 

Or I may add this one of these days but it's not in my priorities...
